src/lib.rs

```diff
@@ -466,6 +467,7 @@ mod tests {
         assert_eq!(&input, output2);
     }
 
+    #[cfg(feature = "std")]
     #[test]
     fn u16() {
```

Comment:
Can you split this test so that we can run the main part on no_std and have the part that needs std separately?

Reply:
Done. Is such test name OK?